scalafix-core/shared/src/main/scala/scalafix/rewrite/RewriteCtx.scala

```diff
@@ -77,5 +78,10 @@ case class RewriteCtx(tree: Tree, config: ScalafixConfig) extends PatchOps {
   def replaceSymbol(fromSymbol: Symbol.Global, toSymbol: Symbol.Global)(
       implicit mirror: Mirror): Patch =
     TreePatch.ReplaceSymbol(fromSymbol, toSymbol)
+  def renameSymbol(fromSymbol: Symbol.Global, toName: String)(
+      implicit mirror: Mirror): Patch =
+    TreePatch.ReplaceSymbol(
```

Comment:
We may later want to introduce proper node for renames instead of re-using ReplaceSymbol. This won't affect the public API however since it's an implementation detail.